package.json

```
@@ -63,7 +63,7 @@
     "gray-matter": "^4.0.2",
     "jsdom": "^16.3.0",
     "node-sass": "^4.14.1",
-    "platformicons": "^4.1.4",
+    "platformicons": "^4.1.6",
```

Bumping will happen here (to 4.1.5: https://github.com/getsentry/sentry-docs/pull/3387)

```suggestion
    "platformicons": "^4.1.4",
```